platform/core/flatbuffers_shared/generate_flatbuffers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+echo '===> Compiling the Schema...'
+
+cd flatbuffers_shared
+# TODO: Add iOS and Android folders
+RUST_FLATBUFFERS_OUT=../src/flatbuffers_generated
+FLATBUFFER_MODELS=$( find -P flatbuffer_models -name "*.fbs" )
+
+echo "*** Removing previously compiled models ***"
+rm -rf $RUST_FLATBUFFERS_OUT && mkdir $RUST_FLATBUFFERS_OUT
+
+echo "*** Compiling models ***"
+for flatbuffer in $FLATBUFFER_MODELS; do
+    ./flatc --rust -o ${RUST_FLATBUFFERS_OUT} --include-prefix flatbuffers_generated "$flatbuffer"
+    # Generate import from `./flatbuffer_models/flatbuffer_name.fbs`.
+    echo "pub mod $(basename "${flatbuffer%.fbs}")_generated;" >> ${RUST_FLATBUFFERS_OUT}/mod.rs
+
+    # TODO: Generate for Swift/Kotlin
+done
+cd ..
+
+echo '===> Done!'
```

Comment:
Also, at the end of the day this is just compilation. Using a Makefile target, with a Makefile rule for generating one from the other would save a bit of boilerplate. That's what make, who is just a tiny bit younger than me, was made for, anyway

Reply:
Makefile was replaced with a shell script due to portability issues, and nothing else.

Reply:
Still, I don't get why installing bash (which is not standard in Windows) should be preferred to installing `make`.

Reply:
As far as I know, bash will most probably be configured when one wants to have `git` on Windows. And I see that make is used currently only for iOS builds. Thus, if one want's to build core, they will not need to additional config such as making sure to have `make` installed on the Windows.

Even though I am new to Makefile, I am in favour of using it. It seems to be possible to install `make`
with Chocolatey(idk how widely this is used), but I don't know how portable is the content of Makefile itself. I assume we would need to have OS specific commands, for example `rmdir` instead of `rm` on Windows. With Bash we don't have this problem.

Reply:
make can probably be configured to use the existing bash as a shell by default, simply by [setting the `SHELL` variable](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html). You've already used `ifeq` elsewhere, you can use it [like this](https://stackoverflow.com/a/63840549/891440) to set the shell.

Reply:
fair enough, will check that out. Probably I would add a Makefile to manage the execution of the two bash scripts that were added. If one would want's to use make, it will be possible. If not, they can use the bash scripts. Leaving more options available.